docs/tsc/meetings/2019-03-04.md

```diff
@@ -0,0 +1,92 @@
+        - Doug: Testing off of master at any given time is stable, and ideal.
+
+* ASWF/LF CI update from TAC meeting
+    - Thanh presented comparison of Circle CI and current Jenkins infrastructure:
+        - Overall, TAC impression is that Circle seems like a better choice, but pricing and logistics needs further investigation.
+        - Circle supports local CI, allowing contributors to test CI *before* submitting a PR.
+        - Supports GPUs when hosted on our servers, but pricing may be a concern.
+        - Proposed AppVeyor for Windows support for now.
+    - OpenVDB reports that Circle significantly faster than Travis
+    - Larry: Circle has better, faster CPUs than Travis
+    - **TODO**: Once ASWF/LF commit to CI choice, Mark/Sean to investigate transition to new CI infastructure
+        - If OpenVDB adopts new system first, we can reference their setup.
+
+* Moving OCIO to ASWF Git organization
+    - Legal approvals for OCIO domain tranfer are complete!
+    - The GitHub account move should be fairly straightforward, but may bring down CI builds for a day or two diring transition.
+    - **TODO**: LF needs Travis and AppVeyor accounts created before we can move repo, to host CI builds.
+    - **TODO**: Larry/Michael to work with John/Thanh to facilitate the account move.
+    - There may be a requirement to fix some of the embedded external dependencies *before* the repo move to satisfy detected license issues (from the license scan).
+
+* OCIO v2 Python bindings
+    - SPI proposes contributing new bindings via pybind11:
+        - Mark: Current OCIO bindings have no external dependency. Are we sure we want to introduce one, vs. using Python's C API?
+            - Larry: Supporting both Python 2 and 3 would make use of C API more complicated. pybind11 is simple to implement, and header only.
+    - Sean: Good opportunity to improve consistency in Python interface:
+        - Currently there are inconsistencies with argument and return types.
+        - Implementors are not always sure what types they'll get back from the API.
+    - Larry: proposes a design PR be created first (using new GH feature), which outlines the new interface, prior to actually doing the work.
+    - **TODO**: Michael to create new Python API design and submit PR for feedback/discussion/approval.
+
+* Apple has deprecated OpenGL
+    - Mark: One option could be to build a bridget between Vulcan and Metal. Proposes use of MoltenVK.
+    - Sean: SPIRV-Cross also worth investigation.
+    - Doug: We're not sure *when* OpenGL support will actually be removed.
+        - It's a matter of when, not if...
+        - Agrees MoltenVK is a good consideration.
+    - Carl: Different parties within Apple have different opinions of timeline for dropping support.
+    - Mark: Nothing in OCIO's design is preventing Metal support from being added now.
+        - Doug: Adds that per-language texture support is already present as well.
```

Comment:
That's not quite what I said.  OCIO already has the ability to emit different shader code based on language (e.g. GLSL vs HLSL).  The point that I was trying to make was that it is not only a question of the shader language, it is also a question of how to manage textures under a low-level paradigm such as Vulkan/Metal and this will also need investigation.

Also, my statement farther up of "when, not if" was not about when Apple will remove support for OpenGL.  What I meant, was that OCIO supporting low-level graphics APIs including Vulkan and Metal is a question of when, not if.  In other words, there is no doubt that we will need that for our products at some point.

Reply:
So, I'll propose the following summary.  Here you could write:
- Doug: Investigation is needed not only on the shader language but also on how textures will be managed under a low-level graphics API.

And above you could write:
- Doug: We're not sure yet *when* OpenGL support will actually be removed by Apple.
   - From my point of view, OCIO will definitely need to support low-level graphics APIs such as Vulkan and Metal, so the question is more when than if.
   - Agreed MoltenVK is one option to investigate.

Reply:
Thanks for the clarification @doug-walker